src/Package.php

```diff
@@ -22,6 +22,13 @@ class Package
      */
     public $tracking_code;
 
+     /**
+     * Returns Web URL for a specific tracking/carrier
+     *
+     * @var string
+     */
+    public $web_link;
```

Comment:
@bell2041 I think a better name for this property would be `tracking_url` - similar to `tracking_code`.

Reply:
done